tests/CMakeLists.txt

```diff
@@ -25,6 +28,7 @@ link_directories("${source_dir}/libs/gherkin-cpp")
 
 set(SOURCES
     connection_pool.cpp
+    type_traits.cpp
```

Comment:
@thed636 I think you force-pushed the wrong commit as head of the branch, I don't see this file or anything actually related to traits in the PR

Reply:
Oops, my mistake. I've pushed the branch one more time.